conf.py

```diff
@@ -203,9 +200,3 @@ def setup(app):
 
 locale_dirs = ['locale/']   # path is example but recommended.
 gettext_compact = False     # optional.
-
-
-## Gen menu
-import gensidebar
```

Comment:
As stated in another comment, I removed it because I noticed that the result of this script was not included anymore in the documentation. If it's used outside of the docs, I'll revert my change with pleasure.

Reply:
If I recall it runs only if you remove the "toctree" cache

Reply:
https://github.com/openimis/openimis_docs/blob/d8283a3392b6289f78bc03e8833c960a505d04c6/conf.py#L209

Reply:
I've put it back.